### PR TITLE
ACC-545 Contract Details page update

### PR DESCRIPTION
**Pull request by thurstontye**
Updates contract.content_allowed to contain either Article or Rich Article.

This is a draft PR until an example of a saleforce 'rich article' account it's not clear if the changes made here are the correct thing to do.

### CODE REVIEW

File: server/helpers/has-rich-article.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = exports = (prodList= []) => {
+	return (prodList.length >= 1) ? prodList.some(product => product.code === 'S2'): false;
+};
```
Comment (rosario) [inline server/helpers/has-rich-article.js:5]: My 2p, if this function is not used anywhere else then it doesn't need to be into a separate file itself.
In particular if it's such simple one line function.

Comment (rosario) [reply server/helpers/has-rich-article.js:5]: A note about `S2`, in master it's defined as a constant now:

https://github.com/Financial-Times/next-syndication-api/blob/master/config/default.yaml#L246

Comment (thurstontye) [reply server/helpers/has-rich-article.js:5]: @rosario just realised there is a bunch of changes in here that are no longer relevant. Hold fire on reviewing this I'm going to tidy up the PR

**Comment by thurstontye** [comment]
Out of date changes new PR on it's way

**Pull request closed without merging**